docker/web/development/Dockerfile

```diff
@@ -63,7 +63,7 @@ COPY --chown=www-data:www-data ./docker/web/development/docker-entrypoint.sh ./d
 RUN chmod 0775 ./docker-entrypoint.sh
 
 COPY --chown=www-data:www-data . /code
-RUN chmod 0775 ./orm
+RUN chmod 0775 ./orm ./web
```

Comment:
Interesting that we not have this in production or in GEWISWEB (not even for `./orm`).

Reply:
Turns out, `git ls-files --stage web` agrees with you that it should already be an executable file, but I don't think Windows allows this; I think it is useful to have it included at least on Windows (where file permissions aren't really being used by Docker)